dsls/sql/src/main/java/org/apache/beam/dsls/sql/interpreter/operator/arithmetic/BeamSqlArithmeticExpression.java

```diff
@@ -80,12 +80,10 @@ public BeamSqlArithmeticExpression(List<BeamSqlExpression> operands) {
 
   private double getDouble(BeamSqlRow inputRecord, BeamSqlExpression op) {
     Object raw = op.evaluate(inputRecord).getValue();
-    Double ret = null;
     if (SqlTypeName.NUMERIC_TYPES.contains(op.getOutputType())) {
-      ret = ((Number) raw).doubleValue();
+      return ((Number) raw).doubleValue();
     }
-
-    return ret;
+    return 0d;
```

Comment:
For the error with Hamcrest I have the same issue and it was only solved until I did a full `mvn clean install` from the beam parent directory, then the execution worked flawlessly (capricious Maven).
For a quick execution I did:
```
mvn clean install  -DskipTests -Dcheckstyle.skip=true -Denforcer.skip=true -Dmaven.javadoc.skip=true -pl '!sdks/python' -pl '!sdks/java/maven-archetypes/examples-java8' -pl '!sdks/java/maven-archetypes/starter' -pl '!sdks/java/maven-archetypes/examples'
```